ci-operator/config/openshift/operator-framework-tooling/openshift-operator-framework-tooling-main.yaml

```diff
@@ -0,0 +1,63 @@
+base_images:
+  commitchecker:
+    name: commitchecker
+    namespace: ci
+    tag: "4.14"
+  ocp_builder_rhel-8-golang-1.20-openshift-4.15:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.20-openshift-4.15
+  os:
+    name: centos
+    namespace: origin
+    tag: stream8
+binary_build_commands: GO111MODULE=on GOPROXY=https://proxy.golang.org make build
+build_root:
+  from_repository: true
+images:
+- dockerfile_path: v0.Dockerfile
+  from: os
+  inputs:
+    bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/src/github.com/openshift/operator-framework-tooling/v0
```

Comment:
Is this right or should it be cmd/v0?

Reply:
The built binary lands at the repo root from `go build` so this is right